lib/matplotlib/colors.py

```diff
@@ -557,8 +557,6 @@ def __call__(self, X, alpha=None, bytes=False):
                 # override its alpha just as for any other value.
 
         rgba = lut.take(xa, axis=0, mode='clip')
-        if vtype == 'scalar':
```

Comment:
Won't removing this change the return type of `cm(.5)`?

Reply:
Yes, good catch! I apparently neglected to read the return statement of the method requiring a tuple if the input is a scalar.
Old:  (0.798216, 0.280197, 0.469538, 1.0)
New: array([0.798216, 0.280197, 0.469538, 1.0])

I was trying to clean the code up a bit and not have multiple if statements for the two different cases. A simple fix to keep things consistent would be:
```
if np.iterable(X):
    # Return a tuple if the input is scalar
    rgba = tuple(rgba)
```

I should be able to push a new change with this addition up later today